rpcs3/Emu/Cell/Modules/libmixer.cpp

```diff
@@ -6,6 +6,13 @@
 #include "cellAudio.h"
 #include "libmixer.h"
 
+#if __GNUC__ >= 6
+#include <math.h>
+#define FABS(X) std::fabs(X)
```

Comment:
btw, why do you reopened and closed your pr? Build bots should trigger after each new commit even if it's a rebase.